docs/dqx/docs/guide/summary_metrics.mdx

```diff
@@ -0,0 +1,458 @@
+    ```
+  </TabItem>
+</Tabs>
+
+#### Writing Metrics to a Table with Streaming
+
+Summary metrics can also be written in streaming. When the input data is read as a streaming source, metrics will be written for each streaming micro-batch:
+
+<Admonition type="warning" title="Supported methods">
+Metrics are not directly accessible from the returned Spark Observation when data is processed with streaming. You must use end-to-end methods that persist the output in tables after quality checks are applied such as `apply_checks_and_save_in_table`, `apply_checks_by_metadata_and_save_in_table`, `apply_checks_and_save_in_tables`.
+</Admonition>
+
+<Tabs>
+  <TabItem value="Python" label="Python" default>
+    ```python
+    import time
+    from databricks.labs.dqx import check_funcs
+    from databricks.labs.dqx.engine import DQEngine
+    from databricks.labs.dqx.metrics_observer import DQMetricsObserver
+    from databricks.labs.dqx.rule import DQRowRule, DQDatasetRule
+    from databricks.labs.dqx.config import InputConfig, OutputConfig
+    from databricks.sdk import WorkspaceClient
+
+    # Define the checks
+    checks = [
+      DQRowRule(
+        criticality="warn",
+        check_func=check_funcs.is_not_null,
+        column="col3",
+      ),
+      DQRowRule(
+        name="email_invalid_format",
+        criticality="error",
+        check_func=check_funcs.regex_match,
+        column="email",
+        check_func_kwargs={"regex": r"^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$"},
+      ),
+    ]
+
+    # Create the observer with unique name between runs to make sure metrics are cleaned after each run
+    observer = DQMetricsObserver(name=f"dq_metrics_{int(time.time())}")
+
+    # Create the engine with the metrics observer
+    engine = DQEngine(WorkspaceClient(), observer=observer)
+
+    # Create the input config for a streaming data source
+    input_config = InputConfig("main.demo.input_data", is_streaming=True)
+
+    # Create the output, quarantine, and metrics configs
+    output_config = OutputConfig(
+      location="main.demo.valid_data",
+      trigger={"availableNow": True},  # stop the stream once all data is processed
+      options={"checkpointLocation": "/tmp/checkpoint/valid_data"}  # use a Volume in production for persistence
+    )
+    quarantine_config = OutputConfig(
+      location="main.demo.quarantine_data",
+      trigger={"availableNow": True},  # stop the stream once all data is processed
+      options={"checkpointLocation": "/tmp/checkpoint/quarantine_data"}  # use a Volume in production for persistence
+    )
+    metrics_config = OutputConfig("main.demo.metrics_data")  # streaming configuration not required for metrics
+
+    # Read the data, apply the checks, write data to valid and quarantine tables, and write metrics to the metrics table
+    # Output and quarantine data will be written in streaming and summary metrics will be written for each micro-batch
+    engine.apply_checks_and_save_in_table(
```

Comment:
metrics are written twice when using streaming with output and quarantine. Correct output is written to metrics only when no quarantine is configured.